full/Dockerfile

```diff
@@ -116,6 +108,12 @@ RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.t
 ENV GOPATH=/workspace:$GOPATH \
     PATH=/workspace/bin:$PATH
 
+### Gradle & Maven ###
```

Comment:
This should go into the java section as AFAIU it is not primarily for maven/gradle but for JDKs. I.e. the rvm/nvm of the Java world.

Reply:
Sure, I can move it into the `### Java ###` section above, but I'd need to enclose it in:

```bash
USER gitpod
...
USER root
```